std/range/package.d

```diff
@@ -2887,13 +2905,37 @@ private:
     UT _value;
 
 public:
-    @property inout(T) front() inout { return _value; }
-    @property inout(T) back() inout { return _value; }
+    ///
+    @property inout(T) front() inout
+    {
+        return _value;
+    }
+
+    ///
```

Comment:
If you are going to document, please put something.

I think it's fine to do "Range primitives" on the first one, and then ditto the rest. As of now, the empty docs take up a LOT of space: http://dtest.thecybershadow.net/artifact/website-fc52239a89c0f31ce8fab3844f156b775682a9e6-929fb93b8eac814d30aee0920e8c35b9/web/phobos-prerelease/std_range.html#.Repeat